src/otx/algo/instance_segmentation/segmentors/two_stage.py

```diff
@@ -226,19 +226,44 @@ def export(
         self,
         batch_inputs: torch.Tensor,
         batch_img_metas: list[dict],
-    ) -> tuple[torch.Tensor, ...]:
-        """Export for two stage detectors."""
-        x = self.extract_feat(batch_inputs)
+        explain_mode: bool,
+    ) -> tuple[torch.Tensor, torch.Tensor, torch.Tensor] | dict:
+        """Export the model for ONNX/OpenVINO.
+
+        Args:
+            batch_inputs (torch.Tensor): image tensor with shape (N, C, H, W).
+            batch_img_metas (list[dict]): image information.
+            explain_mode (bool): whether to return feature vector.
 
+        Returns:
+            tuple[torch.Tensor, torch.Tensor, torch.Tensor] | dict:
+                - bboxes (torch.Tensor): bounding boxes.
+                - labels (torch.Tensor): labels.
+                - masks (torch.Tensor): masks.
+                - feature_vector (torch.Tensor, optional): feature vector.
+                - saliency_map (torch.Tensor, optional): saliency map.
```

Comment:
Could you add an additional description of these properties being about dict output?

Reply:
Updated.